examples/basic.html

```diff
@@ -31,6 +31,11 @@
     <section id='sotd'>
       <p>
         This is required.
+        <pre>
```

Comment:
Please remove this. It's ok to use basic.html for your own testing, but no need to modify the basic.html file.

Reply:
Don’t forget to remove this from basic.html